### DISCUSSION TITLE
Collect DNS Name of a Virtual Machine

### DISCUSSION THREAD

**Discussion by AnComet** [bb-Ricardo/netbox-sync#211] [Q&A] [answered]
Hi,

Is there any way to collect the "DNS Name" field of a Virtual Machine in a Custom Field something else in NetBox ?
I actually can't able to make a dns lookup of the different IP Address, but the field exist in the vSphere Client (like in the screenshot "DNS Name (1)")

![Screenshot vSphere](https://user-images.githubusercontent.com/39909865/183955819-a0bb7c78-3c29-4973-9d2e-9271fa75e813.png)

Thank you

**Comment by AnComet** [comment, marked as answer]
Finally I've found how to do that with this configuration in the settings.ini file :
vm_custom_object_attributes = guest.hostName

**Reply by bb-Ricardo** [reply]
Great that you found a solution.

sorry for not getting back, I somehow missed this question.
